widgetssdk/src/main/java/com/glia/widgets/callvisualizer/EndScreenSharingView.kt

```diff
@@ -136,6 +126,7 @@ class EndScreenSharingView (
             textColor = baseDarkColor,
             textFont = fontFamily
         )
+        applyRemoteTheme(Dependencies.getGliaThemeManager().theme)
```

Comment:
I think it’s better to move this line to `init{}` block, because it’s not a part of remote theme

Reply:
Changed. Besides, refactored the `init()` function to make it more readable and clear.